doc/sphinx/proof-engine/vernacular-commands.rst

```diff
@@ -608,11 +608,11 @@ file is a particular case of module called *library file*.
 
       This loads and declares the module :n:`@qualid`
       and its dependencies then imports the contents of :n:`@qualid` as described
-      :ref:`here <import_qualid>`. It does not import the modules on which
-      qualid depends unless these modules were themselves required in module
+      for :cmd`Import`. It does not import the modules that
+      :n:`@qualid`` depends on unless these modules were themselves required in module
       :n:`@qualid`
-      using :cmd:`Require Export`, as described below, or recursively required
-      through a sequence of :cmd:`Require Export`.  If the module required has
+      using :cmd:`Require Export`, or recursively required
+      through a series of :cmd:`Require Export`.  If the module required has
       already been loaded, :cmd:`Require Import` :n:`@qualid` simply imports it, as
       :cmd:`Import` :n:`@qualid` would.
```

Comment:
This could be simplified to:

```suggestion
      already been loaded, :n:`Require Import @qualid` simply imports it, as
      :n:`Import @qualid` would.
```

Reply:
Btw, are we not using ``:token:`xyz` `` anymore? I guess it's redundant given that we have ``:n:`@xyz` ``

Reply:
Yes, at some point there was some confused discussion about this and in the end Jim homogenized most of the sources to use `:n:` everywhere.

Reply:
IIRC the discussion was initially confused; we agreed they were equivalent for your example.  I am using `:n:` in modified text because it can do more (as in the examples above) and simpler to use one construct instead of two.  I'm not going out my way to change `:token:` to `:n:` (except maybe when the OCD kicks in :-).  There are many uses of `:token:` (45 in this file, over 1000 across the doc).